news/2018-05-16-twc-finals-recap.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "osu!taiko World Cup - Finals Recap!"
+date: 2018-05-16 00:00:00 +0000
+---
+The drum mashing of the osu!taiko World Cup 2018 has concluded. The journey for the gold medal has left us with one winning team, but definitely not in the dominant fashion that was expected. 
+
+![](/wiki/shared/news/2018-05-07-twc-finals-recap/podium.png)
+
+**Note: Due to an unfortunate situation surrounding foul play, team United States has forfeited out of the TWC. We won't go over their matches in this recap. You can read about the situation [in this thread](https://osu.ppy.sh/community/forums/topics/743380).**
```

Comment:
```diff
- recap. You
+ recap, but you
```
The second to last sentence and the last sentence could be one connected by a conjunction...?